src/features/repo-setup/model/user-repo-setup.ts

```
@@ -0,0 +1,183 @@
+  }, [apiFiles]);
+
+  const handleToggleSelection = useCallback((nodeId: string, nodeData: FileNode) => {
+    const idsToToggle = collectAllIds(nodeData);
+
+    setSelectedIds((prev) => {
+      const newSelection = new Set(prev);
+      const currentState = getFolderSelectionState(nodeData, prev);
+      const isCurrentlySelected = currentState === true || currentState === "indeterminate";
+
+      if (isCurrentlySelected) {
+        idsToToggle.forEach((id) => newSelection.delete(id));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)